prisma/schema.prisma

```diff
@@ -186,6 +221,10 @@ model User {
   auditsCreatedBy   Audit[]   @relation(name: "createdBy")
   auditsCreatedFor  Audit[]   @relation(name: "createdFor")
   branchesCreatedBy Branch[]  @relation(name: "createdBy")
+  pullRequestsCreatedBy PullRequest[] @relation(name: "createdBy")
+  pullRequestsMergedBy PullRequest[] @relation(name: "mergedBy")
+  pullRequestsClosedBy PullRequest[] @relation(name: "closedBy")
```

Comment:
Lets rename these to, for readability 👇🏼 

`closedPRs` `mergedPRs` `createdPRs`

Reply:
Okay